Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Используем официальный образ Python
+FROM python:3.8-slim AS build
+
+# Устанавливаем переменные окружения
+ENV PYTHONUNBUFFERED=1
+
+# Создаем рабочую директорию
+WORKDIR /app
+
+# Копируем только requirements.txt для кэширования зависимостей
+COPY requirements.txt /app/
+
+# Устанавливаем зависимости
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Копируем весь проект в контейнер
+COPY . /app/
+
+# Выполняем миграции (при сборке базы данных может не быть, удаляем этот шаг)
+# RUN python manage.py migrate
+
+# Указываем порт, который будет использовать приложение
+EXPOSE 8080
+
+# Команда для запуска приложения
+CMD ["sh", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8080"]
```

Comment:
Using `sh -c` is unnecessary unless you need shell features. You can directly specify the command as `CMD ["python", "manage.py", "migrate", "&&", "python", "manage.py", "runserver", "0.0.0.0:8080"]` if you want to run both commands sequentially. However, note that the `migrate` command might fail if the database is not ready. Consider handling migrations separately.